package.json

```diff
@@ -4,7 +4,7 @@
   "description": "alpha team project",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test nyc --reporter=text --reporter=lcov mocha  --require ts-node/register   test/server-run.ts --coverage",
+    "test": "cross-env NODE_ENV=test nyc --reporter=text --reporter=lcov mocha  --require ts-node/register   test/*.ts --coverage",
```

Comment:
I have to ask who the test will run without migration or demigration think about this